From 83fa190f6d8d4a614124eb867855ea4564495865 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Wed, 17 Mar 2021 23:35:54 +0200
Subject: [PATCH] [SW-0] make: hl-fw-loader: support params reconfig

From time to time there is a need to change sub project's
build parameters, mainly for debug and development.

In order to decouple local changes and leave the maintained
code untouched. A -include option, seems to be the must reasonable
way to allow it.

Change-Id: Id06cf9d34ee4a05f589497f937ad6b835d9c51c3
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/package/hl-fw-loader/hl-fw-loader.mk b/package/hl-fw-loader/hl-fw-loader.mk
index 0e9fca6..590fe8d 100644
--- a/package/hl-fw-loader/hl-fw-loader.mk
+++ b/package/hl-fw-loader/hl-fw-loader.mk
@@ -9,4 +9,8 @@
 	$(INSTALL) -s -D -m 0755 $(@D)/bin/hl-fw-loader $(BINARIES_DIR)/hl-fw-loader
 endef
 
+# if file user_overrun.mk exist locally it will be called
+# and allows re-configuring the make parameters.
+-include user_overrun.mk
+
 $(eval $(host-cmake-package))
